### ISSUE TITLE
store.open_data ignores parameter 'bbox'

### ISSUE THREAD

**Issue by AliceBalfanz** [dcs4cop/xcube-cci#24]
When requesting a cci-cube for dataset_ID `'esacci.SST.day.L4.SSTdepth.multi-sensor.multi-platform.OSTIA.1-1.r1'` the parameter bbox is ignored, and the global extent is returned: 

To reproduce: 

```
%matplotlib inline
# CCI Data Store requires asyncio. In order to work in Notebooks, we need to do the following:
import nest_asyncio
nest_asyncio.apply()


from xcube.core.store import find_data_store_extensions
from xcube.core.store import get_data_store_params_schema
from xcube.core.store import new_data_store

store = new_data_store('cciodp')
store

dataset = store.open_data(
    'esacci.SST.day.L4.SSTdepth.multi-sensor.multi-platform.OSTIA.1-1.r1', 
    variable_names=['analysed_sst'],
    bbox=[5, 35, 6, 36],
    time_range=['2008-07-01','2008-07-10'])

dataset


dataset.analysed_sst.isel(time=0).plot.imshow(cmap='plasma')
```
![image](https://user-images.githubusercontent.com/23457217/102100510-cbe07c00-3e29-11eb-934e-cf7bf2bc8bed.png)

**Comment by TonioF** [comment]
If you do not specify the opener, the cci odp data store will return data as a dataset. For datasets, spatial subsetting is not supported (as they might not have the necessary spatial dimensions). To create a subset, you need to specify to open the data as a cube:
dataset = store.open_data(
    'esacci.SST.day.L4.SSTdepth.multi-sensor.multi-platform.OSTIA.1-1.r1', 
    opener_id='dataset[cube]:zarr:cciodp'
    variable_names=['analysed_sst'],
    bbox=[5, 35, 6, 36],
    time_range=['2008-07-01','2008-07-10'])

(In your example, you only get a white area, so I recommend to use another bounding box to see the effect.)

This does not mean that there is no bug here, though. Calling 
store.get_open_data_params_schema() should tell you that you have to use the global bounding box extent, but it doesn't.

**Comment by TonioF** [comment]
Will be solved by #34 .

**Comment by TonioF** [comment]
Aolved by #34
